packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -600,6 +600,19 @@ const ComboBox = forwardRef(
                   event.target.value.length
                 );
               }
+
+              if (event.altKey && event.key == 'ArrowDown') {
+                event.preventDownshiftDefault = true;
+                if (!isOpen) {
+                  toggleMenu();
+                }
+              }
+              if (event.altKey && event.key == 'ArrowUp') {
+                event.preventDownshiftDefault = true;
```

Comment:
What does this prevent exactly?

Reply:
It prevents the [default behavior](https://github.com/downshift-js/downshift/blob/master/README.md?plain=1#L952C1-L957C72) of `ArrowUp` button, usually default behaviour of `ArrowUp` is not shown when both button are pressed carefully but  some time I have detected this behavior